src/app/shared/components/template/components/layout/display-group/display-group.component.scss

```diff
@@ -89,3 +89,13 @@
     --border-color: var(--border-color-default, var(--ion-color-primary));
   }
 }
+
+.display-group-wrapper {
+  &[data-variant~="solid_secondary"] {
+    padding: var(--regular-padding);
+    background-color: var(--ion-color-secondary);
```

Comment:
nit(non-blocking): when using `box_secondary` the background is `var(--ion-color-secondary-200)` but when using `solid_secondary` we have `var(--ion-color-secondary)`. Even if foregoing the possible confusion of what is a box vs what is solid, there's not really any indication of why one variant uses lighter colours and other darker (unless that is the distinction?)

I think it might be better to try and define either within the existing box semantics, e.g. `box_secondary_alt` if we're only changing a few of the colour weights, or again just overriding at the deployment level

Reply:
Good point. I thought I needed to handle separately to avoid the MX theme level overrides (the `!important` flags, for example, can make it difficult to go back and change styling at the component level). But I've opted for `box_secondary_alt` within the existing box semantics, and it works well with more minimal new style rules. Also addresses your other inline comment. 

See [new commits](https://github.com/IDEMSInternational/open-app-builder/pull/2773/files/5f653b14e36b45da7b2cd9147ebea304abd2ee9c..1cfd2470288a05ce04f8d60493f35cebf8b3ed30).